docs/src/examples/QSelect/InputValueEvent.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="q-pa-md">
+    <div :style="inputVal? '' : 'padding-bottom:21px;' ">
+      {{inputVal}}
+    </div>
+    <div class="q-gutter-md row">
+      <q-select
+        filled
+        v-model="model"
+        use-input
+        hide-selected
+        fill-input
+        input-debounce="0"
+        @input-value="updateInputValue"
```

Comment:
Isn't it what filterFn gives you? 
```
    // val is what the user types
    filterFn (val, update, abort) {
      update(() => {
        const needle = val.toLowerCase()
        this.options = stringOptions.filter(v => v.toLowerCase().indexOf(needle) > -1)
      })
    },
```

Reply:
Seems to be the same in a more direct approach. The actual way seems to be more free in the way of sorting or filtering. Cause with the filterFn you can do basically anything with the array of data.

I really can't understand the meaning of this repo. Can you give us some light?